src/web/dialog.js

```diff
@@ -1,4 +1,68 @@
-let totalCheckboxCount = 0;
+function parse(recipient) {
```

Comment:
`parse` and `RecipientClassifier` are copied from `.mjs` files.
This is because adding

```
<script type="module" src="recipient-classifier.mjs"></script>
<script type="module" src="recipient-parser.mjs"></script>
```

to `dialog.html` raises an error like below.

```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.
```

I think we should fix web server implementation so as to specify "javascript", but I think it is better to fix it in another PR in order to narrow the scope of modification.